11-optimistic-ui/bee-rich/README.md

```diff
@@ -331,8 +333,6 @@ Follow the same pattern to rename the helper functions in `app/routes/dashboard/
 Replace the existing database query in the `handleDelete` helper function in `app/routes/dashboard/expenses/$id/index.tsx`.
 
 ```tsx
-import { deleteExpense } from "~/server/expenses.server";
```

Comment:
I suggested removing this line just because you don't include the import lines in any of the other code examples in this file